config/default.json

```diff
@@ -64,5 +64,15 @@
   },
   "encrypt": {
     "key": "some key here it should probably be really really long"
+  },
+  "oauth_apps": {
+    "github": {
+      "client_secret": "<Contact oauth app holder for this secret>",
+      "client_id": "6b57706945fa18ee0397"
```

Comment:
I think you can leave the `client_id` for both of these oauth apps blank, correct? It's just supposed to be a template, so I think we would leave these out and just have the user enter them in the `production.json` or `test.json`

Reply:
agreed. We really should follow the pattern `default-example.json` and ignore this file. Since most of us use this for development, it should really be getting ignored. However, this is outside the scope of this issue.